modules/consul-cluster/main.tf

```diff
@@ -135,7 +89,21 @@ resource "google_compute_instance_template" "consul_server_private" {
   network_interface {
     network            = var.subnetwork_name != null ? null : var.network_name
     subnetwork         = var.subnetwork_name != null ? var.subnetwork_name : null
-    subnetwork_project = var.network_project_id != null ? var.network_project_id : var.gcp_project_id
+    subnetwork_project = var.network_project_id != null ? var.network_project_id : null
+
+    #access_config {
+    # The presence of this property assigns a public IP address to each Compute Instance. We intentionally leave it
+    # blank so that an external IP address is selected automatically.
+    #  nat_ip = ""
+    #}
+
```

Comment:
Intentional?

Reply:
Nope that needs to go, thanks for spotting!